src/hooks/Guilds/useVoteSummary.ts

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react';
+import { useProposal } from 'hooks/Guilds/ether-swr/guild/useProposal';
+import { useGuildConfig } from './ether-swr/guild/useGuildConfig';
+import { getBigNumberPercentage } from 'utils/bnPercentage';
+
+// Gets vote summary as array of percentages
+export default function useVoteSummary(
+  guildId: string,
+  proposalId: string
+): number[] {
+  const [votes, setVotes] = useState<number[]>([]);
+  const { data: { totalVotes = null } = {} } = useProposal(guildId, proposalId);
+  const { data: { totalLocked = null } = {} } = useGuildConfig(guildId);
+
+  useEffect(() => {
+    if (totalVotes && totalLocked) {
+      const getVoteData = async () => {

```

Comment:
Any reason why this is async? If we don't need this to be async we can memoize this nicely.